ctakes-service/src/main/java/org/phenotips/textanalysis/internal/CTakesAnnotationService.java

```diff
@@ -77,10 +75,7 @@ public CTakesAnnotationService()
             XMLInputSource in = new XMLInputSource(engineXML);
             ResourceSpecifier specifier = UIMAFramework.getXMLParser().parseResourceSpecifier(in);
             this.engine = UIMAFramework.produceAnalysisEngine(specifier);
-            GenericObjectPoolConfig config = new GenericObjectPoolConfig();
-            config.setMaxTotal(3);
-            config.setMinIdle(1);
-            this.jcasPool = new GenericObjectPool<>(new CasFactory(this.engine), config);
+            this.jcasPool = new JCasPool(8, this.engine);
```

Comment:
Under high load and with the default memory settings of the standalone distribution, an OutOfMemory will occur. The pool size should be smaller.